assets/charts/control-plane/kube-apiserver/templates/kube-apiserver.yaml

```diff
@@ -1,31 +1,8 @@
-# On platforms like AWS, it's enough for kube-apiserver to listen on Node InternalIP. On other
```

Comment:
The commit message for this commit is no longer valid. We do not run 2 apiservers for single controller node for example.